test/functional/tools/creates_html.xml

```diff
@@ -0,0 +1,14 @@
+<tool id="creates_html" name="creates_html" version="1.0.0">
+  <command>
+    mkdir '$output.extra_files_path';
```

Comment:
Works for me in this case.

On the other hand, I think IUC recommendations will stick with && and CDATA for some time for backward compatibility. Also CDATA is always needed for '>'.